jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/WorkItemNodeInstance.java

```diff
@@ -81,6 +91,8 @@
 
     private long workItemId = -1;
     private transient WorkItem workItem;
+    
+    private long processInstanceId = -1;
```

Comment:
The naming here can be quite confusing since this suggests that this is the processInstanceId of the processInstance which this node belongs to. Can we change it to something like exceptionHandlingProcessInstanceId or something that would tell that this is the id of the process instance which handles handler exceptions?